src/main/java/uk/gov/digital/ho/hocs/info/api/SomuTypeResource.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.digital.ho.hocs.info.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.digital.ho.hocs.info.api.dto.CreateSomuTypeDto;
+import uk.gov.digital.ho.hocs.info.api.dto.SomuTypeDto;
+import uk.gov.digital.ho.hocs.info.domain.model.SomuType;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+@RestController
+public class SomuTypeResource {
+
+    private final SomuTypeService somuTypeService;
+
+    @Autowired
+    public SomuTypeResource(SomuTypeService somuTypeService) {
+        this.somuTypeService = somuTypeService;
+    }
+
+    @GetMapping(value = "/somuType", produces = APPLICATION_JSON_UTF8_VALUE)
+    ResponseEntity<Set<SomuTypeDto>> getAllActiveSomuTypes() {
```

Comment:
For MUI will this work? What if we want to reactivate a somu_type? Be better just to return all and let the frontend handle whether to show the active? Otherwise we have no way of updating and reactivating, if we need too...